config/locales/en/brexit_landing_page.yml

```diff
@@ -28,6 +28,13 @@ en:
         metadata:
           public_updated_at: 2021-06-08
           document_type: Policy paper
+      - link:
+          text: Apply for a grant to help small and medium-sized businesses new to importing or exporting
+          path: "/guidance/grants-to-help-small-and-medium-sized-businesses-new-to-importing-or-exporting"
+          description: The SME Brexit Support Fund could give you up to £2,000 to help with training or professional advice.
+        metadata:
+          public_updated_at: 2021-02-11
```

Comment:
Just to double-check, this is not `2021-04-15` (the last updated date), but the published date.

Reply:
yep, I think they want to emphasise how long it's been available for